src/server/obj-info.c

```diff
@@ -916,8 +977,8 @@ static bool screen_out_head(const object_type *o_ptr)
 			has_description = TRUE;
 		}
 	}
-
-	return (has_description);
+	/* We are done. */
```

Comment:
Irrelevant comment

Reply:
This and the other irrelevant comment have been removed.